Tasks/AzureResourceManagerTemplateDeploymentV3/operations/Utils.ts

```diff
@@ -444,6 +444,22 @@ class Utils {
                 throw new Error(tl.loc("AzureCLINotFound"));
             }
         }
+        
+        if(filePathExtension === 'bicepparam'){
+            var fileName: string = filePath.split(path.sep).pop().split('.')[0];
+            let azcliversion = await this.getAzureCliVersion()
+            if(parseFloat(azcliversion)){
+                if(this.isBicepParamAvailable(azcliversion)){
+                    await this.execBicepParamBuild(filePath, fileName)
```

Comment:
I think we need to login into AZ before doing a bicep build and do logout after build

Reply:
This now happens with the changes that have been pushed.